src/plugin/leaflet.canvas-markers.js

```diff
@@ -24,7 +24,8 @@ function layerFactory(L) {
             this._draw();
         },
         onRemove: function () {
-            this._destroyContainer();
+            // TODO
+            // this._destroyContainer();
```

Comment:
> Call this._destroyContainer() is unique to this plugin and redundant

Currently `L.CanvasMarkerLayer` has dual nature:
- `L.Layer`, from which it's inherited initially.
- And after https://github.com/Spaction/Leaflet.Canvas-Markers/commit/153525e045e1c4ec22a0ecb02d965366e99852a0 it has some features of `L.Renderer` (but still is not inherited from it)

I can guess that mentioned commit was just a step to make true renderer which can be used transparently (#2).
But in it's current (perhaps unfinished) state `L.CanvasMarkerLayer` is rather layer, so it should'n destroy it's container on remove.

I wonder if we've chosen right fork to continue..